tests/utils/shippable/shippable.sh

```diff
@@ -145,11 +145,13 @@ function cleanup
             set -ux
 
             # shellcheck disable=SC2086
-            ansible-test coverage xml --color --requirements --group-by command --group-by version ${stub:+"$stub"}
+            ansible-test coverage xml --color -v --requirements --group-by command --group-by version ${stub:+"$stub"}
             cp -a tests/output/reports/coverage=*.xml "$SHIPPABLE_RESULT_DIR/codecoverage/"
 
-            # analyze and capture code coverage aggregated by integration test target
-            ansible-test coverage analyze targets generate -v "$SHIPPABLE_RESULT_DIR/testresults/coverage-analyze-targets.json"
+            if [ "${ansible_version}" != "2.9" ]; then
+                # analyze and capture code coverage aggregated by integration test target
```

Comment:
Could you possibly add some details regarding why this isn't needed for future releases

Reply:
It's only needed for 2.10 and later releases, since the CLI syntax changed between 2.9 and 2.10 (some parts were moved to a separate subcommand). More changes in the future are only necessary if the CLI syntax changes again.